src/RoleObjects/SingleSelectListBoxData.test.js

```diff
@@ -0,0 +1,235 @@
+import * as createjs from 'createjs-module';
+import AccessibilityModule from '../index';
+import { parentEl, stage, container } from '../__jestSharedSetup';
+
+describe('SingleSelectListBoxData', () => {
+  describe('register role', () => {
+    let cjsListBox;
+    let cjsSpan;
+    let ulEl;
+    let shouldAutoFocus;
+    let shouldEnableKeyEvents;
+    let isEnabled;
+    let isRequired;
+    let nameVal;
+    let sizeVal;
+
+    beforeEach(() => {
+      cjsListBox = new createjs.Shape(); // dummy object
+      cjsSpan = new createjs.Shape(); // dummy child object
+      shouldAutoFocus = true;
+      shouldEnableKeyEvents = false;
+      isEnabled = false;
+      nameVal = 'list_name';
+      sizeVal = 99;
+
+      AccessibilityModule.register({
+        accessibleOptions: {
+          autoFocus: shouldAutoFocus,
+          enabled: isEnabled,
+          enableKeyEvents: shouldEnableKeyEvents,
+          name: nameVal,
+          required: isRequired,
+          size: sizeVal,
+        },
+        displayObject: cjsListBox,
+        parent: container,
+        role: AccessibilityModule.ROLES.SINGLESELECTLISTBOX,
+      });
+
+      AccessibilityModule.register({
+        displayObject: cjsSpan,
+        role: AccessibilityModule.ROLES.SPAN,
+      });
+
+      stage.accessibilityTranslator.update();
+      ulEl = parentEl.querySelector('ul[role=listbox]');
+    });
+
+    describe('rendering', () => {
+      it('creates ul[role=listbox] element', () => {
+        expect(ulEl).not.toBeNull();
+      });
+
+      it('sets "aria-disabled" attribute', () => {
+        ulEl = parentEl.querySelector(
+          `ul[role=listbox][aria-disabled='${!isEnabled}']`
+        );
+        expect(ulEl).not.toBeNull();
+      });
+
+      it('sets "name" attribute', () => {
+        expect(ulEl.getAttribute('name')).toEqual(nameVal);
+      });
+    });
+
+    describe('children checking', () => {
+      const errorObj =
+        /Children of singleselectlistbox must have a role of singleselectoption/;
+      it('throws error attempting to add prohibited child using addChild() ', () => {
+        expect(() => {
+          stage.accessibilityTranslator.update();
+          cjsListBox.accessible.addChild(cjsSpan);
+        }).toThrowError(errorObj);
+      });
+
+      it('throws error attempting to add prohibited child using addChildAt()', () => {
+        expect(() => {
+          stage.accessibilityTranslator.update();
+          cjsListBox.accessible.addChildAt(cjsSpan, 0);
+        }).toThrowError(errorObj);
+      });
+
+      const cjsDummy = new createjs.Shape();
+      AccessibilityModule.register({
+        displayObject: cjsDummy,
+        role: AccessibilityModule.ROLES.SINGLESELECTOPTION,
+      });
```

Comment:
Same as above, let's move these into a `before` so there's never any question when the lines execute.